taxi/__init__.py

```diff
@@ -0,0 +1,2 @@
+def urls(request):
+    return None
```

Comment:
Why dp you need that here?

Reply:
it has to be auto generated, i didnt write this